src/utils.ts

```diff
@@ -1,5 +1,5 @@
 import { Psbt, Transaction } from 'liquidjs-lib';
-import * as messages from 'tdex-protobuf/generated/js/trade_pb';
+import * as messages from 'api-spec/protobuf/gen/js/tdex/v1/trade_pb';
```

Comment:
```suggestion
import * as messages from './api-spec/protobuf/gen/js/tdex/v1/trade_pb';
```

Are sure works without relative path?

Reply:
I bumped tsdx version to 0.14.0. I am not exactly sure if it's fixed in that version.